platform/ui/src/components/FilterWrapper/FilterWrapper.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '@ohif/ui';
+
+const baseLabelClassName =
+  'flex flex-col flex-1 text-white text-lg pl-1 select-none';
+const spanClassName = 'flex flex-row items-center cursor-pointer';
+const sortIconMap = {
+  '-1': 'sorting-active-down',
+  0: 'sorting',
+  1: 'sorting-active-up',
+};
+
+const FilterWrapper = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
+  onLabelClick,
+  className,
+  children,
+}) => {
+  const iconProps = {
+    name: isBeingSorted ? sortIconMap[sortDirection] : 'sorting',
+    className: classnames(
+      'mx-2 w-2',
+      isBeingSorted ? 'text-custom-aquaBright' : 'text-custom-blue'
```

Comment:
CC: @rodrigolabs, have we not addressed colors yet? Any blockers?

Reply:
I created a specific ticket for color naming.